js/navbar/tabColor.js

```diff
@@ -125,10 +125,18 @@ function isLowContrast (color) {
 function adjustColorForTheme (color) {
   // dim the colors late at night or early in the morning, or when dark mode is enabled
   let colorChange = 1
-  if (hours > 20) {
-    colorChange = 1.01 / (1 + 0.9 * Math.pow(Math.E, 1.5 * (hours - 22.75)))
-  } else if (hours < 6.5) {
-    colorChange = 1.04 / (1 + 0.9 * Math.pow(Math.E, -2 * (hours - 5)))
+  let isAuto = false
+
+  settings.listen('darkMode', function(value){
```

Comment:
Since this code only cares about the current value of the setting, and doesn't need to listen for changes, this can just be `settings.get('darkMode')`.

Reply:
Oh, right. Now I've got the idea how it works :) Thanks. Going to change it ASAP